### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by khaled-alnajjar**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (khaled-alnajjar) [inline public/index.html:6]: Its an important meta tag, why to remove it?

File: src/components/DOB/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 import { connect } from "react-redux";
 import "./style.scss";
-import blockImg from "../../images/block-user 1.png";
+import blockImg from "../../assets/images/block-user 1.png";
```
Comment (khaled-alnajjar) [inline src/components/DOB/index.js:4]: the images shouldn't have a number and there should be no white space

File: src/components/common/monthPickerWrapper.js [hunk 3]
```diff
@@ -25,7 +25,13 @@ const MonthPickerWrapper = (props) => {
           caps={false}
           required={props.required || false}
           disabled={props.disabled || false}
-          onChange={(month) => setValueHandler(month - 1 + 2 + "")}
+          onChange={(month) => {
+            if (month) {
+              setValueHandler(month - 1 + 2 + "");
```
Comment (khaled-alnajjar) [inline src/components/common/monthPickerWrapper.js:30]: Why you subtract 1 then adding 2?

File: src/components/footer/index.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import React, { useEffect } from "react";
 import { connect } from "react-redux";
 import "./style.scss";
-import twitter from "../../images/twitter.png";
-import linked from "../../images/linked in.png";
+import twitter from "../../assets/images/twitter.png";
+import linked from "../../assets/images/linked in.png";
```
Comment (khaled-alnajjar) [inline src/components/footer/index.js:5]: remove the white space

File: src/components/recovery/Recovery.jsx [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+      let locationSeconds = location.state.seconds;
+      if (locationSeconds > 0) {
+        setSeconds(locationSeconds);
+      }
+      if (locationMin !== 0) {
+        setMin(locationMin);
+      }
+    } else {
+      if (resendStorage === "third" && reduxMin === 0 && reduxSeconds === 0) {
+        setMin(15);
```
Comment (khaled-alnajjar) [inline src/components/recovery/Recovery.jsx:60]: add the time as a constant global variable in the function scope

File: src/components/recovery/Recovery.jsx [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+    checkResend()
+  }, []);
+
+  
+
+  useEffect(() => {
+    const changeTimer =()=> {
+      if (seconds < 0 && min > 0) {
+        setMin((min) => min - 1);
+        setSeconds(59);
```
Comment (khaled-alnajjar) [inline src/components/recovery/Recovery.jsx:81]: add the time as a constant global variable in the function scope

File: src/components/recovery/Recovery.jsx [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+  };
+
+
+
+  return (
+    <>
+      <div className="form_container recovery_container">
+        <div className="form_wrapper">
+          <h1 className="form_title mb-8">
+            {Localization.title}, {props.userReducer.firstname || "client"}!{" "}
```
Comment (khaled-alnajjar) [inline src/components/recovery/Recovery.jsx:126]: Remove !{" "}, if you need a space add a margin right

File: src/components/verification/Verification.jsx [hunk 8]
```diff
@@ -2,110 +2,67 @@ import React, { useState, useEffect } from "react";
...
+} from "../../shared/storage";
+import resendCode from "./resendCode"
 
 let interval;
-const CodeVerification = (props) => {
-  const user_obj = props.userReducer;
-  const recovery = props.recovery || user_obj.recovery || "01012345678";
+const Verification = (props) => {
+  const userObj = props.userReducer;
+  const recovery = userObj.recovery || props.recovery  || "01012345678";
```
Comment (khaled-alnajjar) [inline src/components/verification/Verification.jsx:18]: Why do you need a static phone number?

File: src/components/verification/Verification.jsx [hunk 9]
```diff
@@ -2,110 +2,67 @@ import React, { useState, useEffect } from "react";
...
     }
-    if (getResend() === "second" && reduxMin === 0 && reduxSeconds === 0) {
+    if (getResend() === "second" ) {
       setMin(1);
       setSeconds(0);
     }
 
-    if (getResend() === "third" && reduxMin === 0 && reduxSeconds === 0) {
+    if (getResend() === "third" ) {
       setMin(15);
```
Comment (khaled-alnajjar) [inline src/components/verification/Verification.jsx:82]: Add as a global variable

File: src/components/verification/resendCode.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    setSeconds((seconds) => seconds - 1);
+  }, 1000);
+
+  if (getResend() === "first") {
+    setResend("second");
+    setMin(1);
+    setSeconds(0);
+  } else if (getResend() === "second") {
+    setResend("third");
+    setMin(15);
```
Comment (khaled-alnajjar) [inline src/components/verification/resendCode.js]: Add as a global variable

File: src/components/verification/resendCode.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+    setSeconds(0);
+    setError(() => formValidation.resend.wait);
+  } else if (getResend() === "third") {
+    setMin(15);
```
Comment (khaled-alnajjar) [inline src/components/verification/resendCode.js]: Add as a global variable

File: src/components/recovery/Recovery.jsx (hunk 7 above)
Comment (DeyaaMutleq) [reply src/components/recovery/Recovery.jsx:126]: That's add by default from prettier sometimes I will remove it

**Pull request merged**